src/test/java/org/springframework/data/redis/core/types/ExpirationUnitTests.java

```diff
@@ -53,4 +56,38 @@ void fromMinutes() {
 		assertThat(expiration.getExpirationTime()).isEqualTo(5L * 60);
 		assertThat(expiration.getTimeUnit()).isEqualTo(TimeUnit.SECONDS);
 	}
+
+	@Test
+	public void equalValuedExpirationsAreEqual() {
```

Comment:
Nit: Test methods can be package-private.

Reply:
Yes